plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/OpenShiftCommonImages.java

```diff
@@ -39,5 +39,11 @@
 	public static final ImageDescriptor SYSTEM_PROCESS = repo.create("systemprocess.gif"); //$NON-NLS-1$ 
 	public static final Image SYSTEM_PROCESS_IMG = repo.getImage("systemprocess.gif"); //$NON-NLS-1$ 
 
+	public static final ImageDescriptor WARNING = repo.create("pficon-warning.png"); //$NON-NLS-1$ 
```

Comment:
Why use a custom warning icon? Normally there's no need for to use a warning image, it's already present in the Eclipse platform. Furthermore I would not differ from the standard.